release-notes.html

```diff
@@ -65,14 +65,15 @@ <h2>How to Run </h2>
 </ol>
 
 <h2>Fixes and Improvements</h2>
-<p>All the fixes and improvements in @product.name@ are listed at the following locations:</p>
+<p>A list of new features and bug fixes shipped with this release are listed at the following locations:</p>
+
 
 <ul>
     <li>
-        <a href="https://wso2.org/jira/issues/?filter=14457">IS Runtime</a>
+        <a href="https://wso2.org/jira/issues/?filter=14514">IS Runtime JIRA</a>
```

Comment:
Shall we change this link to github issues

Reply:
We have added Github issues below. Since we had issues in both we need to have both filters IMO.

Reply:
+1